unity-renderer/Assets/Scripts/MainScripts/DCL/WebInterface/Interface.cs

```diff
@@ -870,8 +870,7 @@ private class GetChannelMembersPayload
     [DllImport("__Internal")] public static extern void StartDecentraland();
     [DllImport("__Internal")] public static extern void MessageFromEngine(string type, string message);
     [DllImport("__Internal")] public static extern string GetGraphicCard();
-    [DllImport("__Internal")] public static extern void BinaryMessageFromEngine(string sceneId, byte[] bytes, int size);
```

Comment:
Hmm probably my mistake, for some reason 4 months ago this was giving me issues so it was removed since no system was using it. 

I'll try plugging it back

Reply:
Yeah, I was right, BinaryMessageFromEngine is correctly implemented at WebGLTransport, the one at the Interface class is not only not used, but it also breaks the compilation because the signature is incorrect (probably only happening on unity 2021).